lnbits/core/views/auth_api.py

```diff
@@ -251,6 +252,36 @@ async def update(
         raise HTTPException(HTTP_500_INTERNAL_SERVER_ERROR, "Cannot update user.")
 
 
+@auth_router.put("/api/v1/auth/first_install")
+async def first_install(data: UpdateSuperuserPassword) -> JSONResponse:
+    if not settings.first_install:
+        raise HTTPException(HTTP_401_UNAUTHORIZED, "This is not your first install")
+    try:
+        assert data.username, "Missing username."
```

Comment:
if `UpdateSuperuserPassword.username` is made not optional, then this `assert` is refundant

Reply:
fixed here: https://github.com/lnbits/lnbits/pull/1977/commits/8a25f65e8f62543a58190a59923c929e3145520b